beacon-chain/sync/validate_data_column.go

```diff
@@ -205,6 +207,15 @@ func (s *Service) validateDataColumn(ctx context.Context, pid peer.ID, msg *pubs
 		log.WithField("slot", roDataColumn.Slot()).Warn("Failed to send data column log entry")
 	}
 
+	if s.cfg.operationNotifier != nil {
+		s.cfg.operationNotifier.OperationFeed().Send(&feed.Event{
+			Type: operation.DataColumnReceived,
+			Data: &operation.DataColumnReceivedData{
+				Data: &verifiedRODataColumns[0].RODataColumn,
```

Comment:
I'm passing the reference here, which might not be safe. I'm assuming the downstream listener won't mutate the data. I'm not sure what the best practice is, and happy to change it to be safer, but that would require copying a list of KZG commitments which is a sane tradeoff